java-logging-appinsights/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+uk.gov.hmcts.reform.logging.appinsights.autoconfiguration.AppInsightsAutoConfiguration
```

Comment:
We don't need that, just stick `ContextVersionInitializer` in [config](https://github.com/hmcts/java-logging/blob/features/appinsightspringstarter/java-logging-appinsights/src/main/resources/ApplicationInsights.xml) and AppInsights should deal with everything for you. Unless that's deprecated too 😯

Reply:
Not sure if using xml is better than java code, we've moved to a spring boot starter, why not use its functionality?

Reply:
Then let's keep one version of configs by completely removing xml :)

Reply:
Done, got rid of XML configs